keps/sig-node/1797-configure-fqdn-as-hostname-for-pods/kep.yaml

```diff
@@ -43,10 +43,9 @@ milestone:
 feature-gates:
   - name: setHostnameAsFQDN
     components:
-      - kubelet
+      - kube-apiserver
```

Comment:
Both kube-apiserver and kubelet right?

[If not - please update also PRR questionaire in README.md]